sceptre/config/graph.py

```diff
@@ -90,15 +90,15 @@ def _generate_edges(self, stack, dependencies):
         :type dependencies: list
         """
         self.logger.debug(
-            "Generate edges for graph {0}".format(self.graph)
+            "Generate edges for graph {0}".format(stack)
         )
         for dependency in dependencies:
-            edge = self.graph.add_edge(dependency, stack)
+            self.graph.add_edge(dependency, stack)
             if not nx.is_directed_acyclic_graph(self.graph):
                 raise CircularDependenciesError(
                     "Dependency cycle detected: {} {}".format(stack,
                                                               dependency))
-            self.logger.debug("Added edge: {}".format(edge))
+            self.logger.debug("Added edge: {}".format(dependency))
```

Comment:
Now I am looking at this line, it might make sense to have something like 

```python 
self.logger.debug("Added dependency: {} to Stack: {}".format(dependency, stack))
```

Reply:
Thanks for your suggestions for the wording. Let's look at two options for the final output
```
Generate dependencies for stack aaa
  Added dependency: bbb
  Added dependency: ccc
Generate dependencies for stack bbb
  Added dependency: ddd
```
```
Added dependency: bbb to Stack: aaa
Added dependency: ccc to Stack: aaa
Added dependency: ddd to Stack: bbb
```
I would prefer the first solution. 
Any other suggestions?

Reply:
First one looks good to me

Reply:
has been added to the PR